starter-code/classes/__tests__/vehicle.test.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const VehicleConstructor = require('../vehicle-constructor.js');
+const { Car, Motorcycle } = require('../vehicle-class.js');
+
```

Comment:
if you wanted  for line 4 you can do almost the same thing you did for line 3, but for classes.